GOVERNANCE.md

```diff
@@ -0,0 +1,76 @@
+# LFESS Governance
+
+LF Energy Standards and Specifications (LFESS) is structured as a Series LLC entity underneath the Joint Development Foundation Projects, LLC entity, a similar legal entity structure used for technical projects hosted at the LF Energy Foundation. Please refer
+
+## Governance structure
+
+The below chart describes the governance structure.
+
+```mermaid
```

Comment:
can the text in this chart link out to the different committees and working groups?

Reply:
That I'm not sure of - need to play more with mermaid.

Reply:
Figured it out - check now.